sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -13,60 +37,33 @@ There were several areas of consistent feedback expressed across the Azure clien
-create a client library that is user-friendly and idiomatic to the JavaScript
-ecosystem.
+### Cross Service SDK improvements
 
-Apart from redesigns resulting from the new [Azure SDK Design Guidelines for Typescript](https://azure.github.io/azure-sdk/typescript_introduction.html#design-principles),
-the latest version improves on several areas from V2:
-
-### Handling backpressure
+The new version of the Event Hubs library also shares some of the cross-service improvements made to the Azure development experience, such as:
 
-Prior to V5, events were delivered as they were received with no ability
-for the user to control the pace. This could result in flooding of downstream
-dependencies as well as confusion about which events had been consumed in
-what order, making checkpointing difficult to do correctly.
+- Using the new `@azure/identity` library to share a single authentication between clients.
+- A unified diagnostics pipeline that offers a common view of the activities across each of the client libraries.
 
-In V5 the model has been simplified so new events are not delivered until the
-previous batch has been consumed by your event handler. You can see a sample
-demonstrating this [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/typescript/src/receiveEvents.ts)
-
-### Specific clients for sending vs receiving
-
-In V5 we've simplified the API surface, making two distinct clients, rather
-than having a single `EventHubClient`:
-
-- [EventHubProducerClient](https://docs.microsoft.com/javascript/api/@azure/event-hubs/eventhubproducerclient)
-  for sending messages.
-- [EventHubConsumerClient](https://docs.microsoft.com/javascript/api/@azure/event-hubs/eventhubconsumerclient)
-  for receiving messages.
-
-We've also merged the functionality from `EventProcessorHost` from the `@azure/event-processor-host` package into
-`EventHubConsumerClient` in the `@azure/event-hubs` package, allowing `EventHubConsumerClient` to be the single
-point of entry for receiving of any type (from single partition, all partitions, or with load balancing and checkpointing features) within Event Hubs.
-
-Important note on checkpoints: `EventHubConsumerClient` does not support legacy checkpoint data.
-i.e. the checkpoints made using the package `@azure/event-processor-host`.
-In order to support interopability between different programming languages, a unifying checkpoint format was needed.
-This and the need to support improvements to the algorithm used for managing partition ownership made breaking changes necessary.
+### New features
 
-### Granular control over retries
+We have a variety of new features in version 5 of the Event Hubs library.
 
-Retry logic and tuning has been externalized, allowing for better configuration
-to better suit your network configuration and reliability.
+- Ability to create a batch of messages with the `EventHubProducerClient.createBatch()` and `EventDataBatch.tryAdd()` APIs.
```

Comment:
How about adding the backpressure point here? From what I recall that was one of the big benefits people have to move to Track 2.  We can add a link to the detailed section that comes later

Reply:
Sure! I was trying to limit how much I 'repeated' myself, but it is a pretty important callout.